apps/researcher/src/app/[locale]/objects/initial-image-fetch-mode.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+
+import {ImageFetchMode, useListStore} from '@colonial-collections/list-store';
+import {Field, Label, Radio, RadioGroup} from '@headlessui/react';
+import {useTranslations} from 'next-intl';
+
+const options = [
+  {translationKey: 'hideImages', value: ImageFetchMode.None},
```

Comment:
To be sure: is the site 'hiding' images, or 'not loading' them? The text says the former, but I would expect the latter.

Reply:
Good point, I will call this 'no images'